src/js/constants.js

```diff
@@ -23,13 +23,19 @@ export const MediaUIEvents = {
 };
 
 export const MediaUIAttributes = {
+  MEDIA_AIRPLAY_UNAVAILABLE: 'media-airplay-unavailable',
+  MEDIA_PIP_UNAVAILABLE: 'media-pip-unavailable',
   MEDIA_PAUSED: 'media-paused',
   MEDIA_MUTED: 'media-muted',
   MEDIA_VOLUME_LEVEL: 'media-volume-level',
   MEDIA_VOLUME: 'media-volume',
+  MEDIA_VOLUME_UNAVAILABLE: 'media-volume-unavailable',
   MEDIA_IS_PIP: 'media-is-pip',
   MEDIA_CAPTIONS_LIST: 'media-captions-list',
   MEDIA_SUBTITLES_LIST: 'media-subtitles-list',
+  // Include this for styling convenience or exclude since it
+  // can be derived from MEDIA_CAPTIONS_LIST && MEDIA_SUBTITLES_LIST? (CJP)
+  MEDIA_CAPTIONS_UNAVAILABLE: 'media-captions-unavailable',
```

Comment:
How would we determine the difference between "unsupported" vs. "unavailable" in this case? Since an empty captions list isn't the same as "unsupported"? Or do we not differentiate?

Reply:
No, we would never be in an "unsupported" case (unless we come up with a scenario that entails "captions can never be used on this platform/playback env").